File: linked_list/examples/createList.py
from .main import Node, LinkedList;

def createList(data1, data2, data3):
	# creates the linked list object
	linkedList = LinkedList();

	# setting the head prop as Node object with data1
	linkedList.head = Node(data1);

	# creating other seprate nodes
	second = Node(data2);
	third = Node(data3);

	# setting the head node link to the second node
	linkedList.head.next = second;

	# setting the link of the second node to the third node
	second.next = third;
	# third node is default set to NULL -> None;
	return linkedList;

File: linked_list/javascript/examples/createList.mjs
import pkg from "../main.js";
const { Node, LinkedList } = pkg;

function createList(data1, data2, data3) {
  let node1, node2, node3;
  node1 = new Node(data1);
  node2 = new Node(data2);
  node3 = new Node(data3);
  const linkedList = new LinkedList();
  linkedList.head = node1;
  node1.next = node2;
  node2.next = node3;
  return linkedList;
}

myList = createList(45, 23, 56);
myList.printList();
